topics/epigenetics/tutorials/cut_and_run/tutorial.md

```diff
@@ -622,7 +603,7 @@ Let's find out the sequence motifs of the TF GATA1. Studies have revealed that G
 > >
 > > > <solution-title></solution-title>
 > > >
-> > > 1. !!!!  Where is this info? !!!
+> > > 1. Where is this info?
```

Comment:
Could not find this info

Reply:
@lldelisle if you have run the tutorial recently could you check. If you go into the Meme-ChiP htlm then click on the DREME link for the GATA motif. Then when you are in the DREME html, click for the GATA motif under more, there it gives you some number how many peaks were positive for the motif. Its fine to give a rough number in the tutroail.